### DISCUSSION TITLE
Tooltip bug with tabs

### DISCUSSION THREAD

**Discussion by snowbollaanm** [zauberzeug/nicegui#1942] [Ideas / Feature Requests]
Hi I have encountered an issue with tooltips and tabs. 

I have made a button that when pressed switches to the next tab panel. However, I gave that button a tooltip and discovered a bug that when the tab changes and you leave your cursor in the position of the button for about half a second the tooltip becomes persistent and no longer goes away until you put your cursor back over the button on the original tab. 

```py
from nicegui import ui

def switchTab():
    tabs = [tabOne, tabTwo]
    currentTab = testTabPanels.value

    if currentTab in tabs:
        changeTab = 1 - tabs.index(currentTab)
        testTabPanels.set_value(tabs[changeTab])

with ui.tabs() as testTabs:
    tabOne = ui.tab(1)
    tabTwo = ui.tab(2)

with ui.tab_panels(testTabs, value=tabOne) as testTabPanels:
    with ui.tab_panel(tabOne):
        ui.label('Tab One')
        with ui.button('switch tab', on_click=lambda: switchTab()):
            ui.tooltip('This is a tooltip').classes('text-body2')
    with ui.tab_panel(tabTwo):
        ui.label('Tab Two')
        ui.button('This button has NO tooltip', on_click=lambda: switchTab())

ui.run()

```
![Screenshot 2023-11-02 134521](https://github.com/zauberzeug/nicegui/assets/139186074/dc39919b-b6d8-46f7-9973-2289b3ec6c94)
![tooltipBug screenshot1](https://github.com/zauberzeug/nicegui/assets/139186074/c2d55819-b678-4e90-848b-608df9eacf17)

**Comment by falkoschindler** [comment]
Thanks for reporting this bug, @snowbollaanm! I created issue #1949.
